### ISSUE TITLE
Polly is being deprecated

### ISSUE THREAD

**Issue by averysmalldog** [averysmalldog/polly#6]
Turns out, sometimes one repo is better than two. I created this repo as an exercise for what I imagined might be some kind of grand, sprawling heap of complex functionality down the line. That's not exactly what happened. Check out the [release note](https://github.com/averysmalldog/tesla-gen3wc-monitor/releases/tag/v0.3.0) for [averysmalldog/tesla-gen3wc-monitor](https://github.com/averysmalldog/tesla-gen3wc-monitor).

Long story short, I'm going to put this repo in read-only mode (archive it) since it is essentially a historical artifact.
